runtime/services/catalog/artifacts/yaml/objects.go

```diff
@@ -14,9 +16,6 @@ import (
 	"github.com/rilldata/rill/runtime/pkg/duration"
 	"github.com/rilldata/rill/runtime/pkg/fileutil"
 	"google.golang.org/protobuf/types/known/structpb"
-
-	// Load IANA time zone data
-	_ "time/tzdata"
```

Comment:
Restore this formatting – not sure why it didn't trigger, but there's supposed to be a CI rule to have blank imports in the third group

Reply:
Haha, looks like it did trigger